bcgov/HOUS-permit-portal
# fix template update notifications and status filters

## Description

Fixed a couple of spots that were still using the old statuses.
Broken notification was causing submitters to not know that their application template was updated and why their application won't submit

## What type of PR is this? (check all applicable)

- [x ] 🐛 Bug Fix

## Related Tickets & Documents

https://hous-bssb.atlassian.net/jira/software/projects/BPHH/boards/1/backlog?assignee=712020%3Ab4c7d328-674e-41ae-b1d1-37b785409ffa&selectedIssue=BPHH-1620&versions=visible

## Steps to QA

create a permit application as submitter
update that permit application's template as a super admin
the submitter should get a notification with a link
the link should navigate the submitter to the permit application index with filters showing the new permit permit application applied
